entries/z/zola.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Zola": {
+    "domain": "zola.com",
+    "tfa": [
+      "sms",
+      "email"
```

Comment:
Please attach some screenshots or other evidence of the MFA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.

Reply:
Btw, where are these screenshots actually used? I don't see them linked from the 2FA Directory search results.

Reply:
We use them to verify that the services have MFA but they are not linked in the directory.